exporter/awsxrayexporter/internal/translator/segment.go

```diff
@@ -155,7 +155,11 @@ func MakeDependencySubsegmentForLocalRootDependencySpan(span ptrace.Span, resour
 	}
 
 	if myAwsRemoteService, ok := span.Attributes().Get(awsRemoteService); ok {
```

Comment:
Not blocking: Would have been good to break this out into a helper to reduce code duplication.

Reply:
Updated with de-duped logic